src/v/config/configuration.cc

```diff
@@ -131,6 +131,14 @@ configuration::configuration()
       {.example = "65536"},
       std::nullopt,
       {.min = 32_KiB, .align = 4_KiB})
+  , rpc_client_connections_per_peer(
+      *this,
+      "rpc_client_connections_per_peer",
+      "The maximum number of connections a broker will open to each of its "
+      "peers",
+      {.example = "8"},
+      8,
+      {.min = 2})
```

Comment:
scared?

Reply:
Very much so, I might even change the min to 8 for this PR.